src/buffer_manager.ts

```diff
@@ -251,14 +252,20 @@ export class BufferManager implements Disposable {
     private async handleOpenFile(data: EventBusData<"open-file">) {
         const [fileName, close] = data;
         const currEditor = window.activeTextEditor;
-        let doc: TextDocument | undefined;
+        let doc: NotebookDocument | TextDocument | undefined;
         try {
             if (fileName === "__vscode_new__") {
                 doc = await workspace.openTextDocument();
             } else {
                 const normalizedName = fileName.trim();
-                const filePath = this.findPathFromFileName(normalizedName);
-                doc = await workspace.openTextDocument(filePath);
+                let uri = Uri.parse(this.findPathFromFileName(normalizedName));
```

Comment:
```suggestion
                let uri = Uri.from({ scheme: "file", path: this.findPathFromFileName(normalizedName) });
```

Reply:
Uri.parse cannot correctly recognize Windows paths, and the drive letter will be treated as a schema.